internal/cli/down.go

```diff
@@ -36,8 +36,8 @@ Deletes resources specified in the Tiltfile
 
 Namespaces are not deleted by default. Use --delete-namespaces to change that.
 
-Local resources (https://docs.tilt.dev/local_resource.html) that are still running (with serve_cmd)
-are not impacted, and need to stopped separately.
+Local resources (https://docs.tilt.dev/local_resource.html) that are still running
```

Comment:
Mmm I think this line is vestigial; since long-running local resources are theoretically NEVER left alive when tilt exits, so this line might be more confusing than helpful.

so there's a type of local resource that ISN'T long-lived, and just runs a single command. it might be useful to clarify:
> `tilt down` has no effect on local resources; let us know if this is functionality you need.

Reply:
Okay, I got this incorrect. So I just reverted. I.e. this PR only changes the tilt up doc.